src/consensus/pbft/pbft.h

```diff
@@ -179,6 +184,19 @@ namespace pbft
         return msg->size();
       }
 
+      if (should_encrypt(msg->tag()))
```

Comment:
This encrypts too much data. We only need to encrypt the payload of the user request and response. This is a performance overhead.